### ISSUE TITLE
Questions about InstagramConstants

### ISSUE THREAD

**Issue by yaniferhaoui** [instagram4j/instagram4j#347]
Hello, recently, my bot is blocked by Instagram without I change his behavior, but my account isn't blocked on my Android APK.

I would like to know if the problem could be that every people here use the same configuration (DEVICE_MANUFACTURER, DEVICE_MODEL, DEVICE_EXPERIMENTS, USER_AGENT....) and so with time, Instagram begin to identify our Bots thanks this datas?

We use an old android version and an old instagram version.. do you think that could be one of the reasons?

One more thing, could we change/customize easily these datas or it's difficult and/or dangerous?

If someone could explain or has infos about that?

Thanks for your time :)

**Comment by jvogit** [comment]
Addressed in [rewrite](https://github.com/instagram4j/instagram4j/tree/develop) or no longer relevant.
